cli/util/api.go

```diff
@@ -274,7 +274,7 @@ func FullNodeProxy[T api.FullNode](ins []T, outstr *api.FullNodeStruct) {
 				}
 
 				total := len(rins)
-				result, err := retry.Retry(ctx, 5, initialBackoff, errorsToRetry, func() (results []reflect.Value, err2 error) {
+				result, _ := retry.Retry(ctx, 5, initialBackoff, errorsToRetry, func() ([]reflect.Value, error) {
```

Comment:
This is just a temporary ignore for the linter... This gas should probably be propagated https://github.com/filecoin-project/lotus/issues/9735